README.md

```diff
@@ -53,7 +53,7 @@ The following features are defined:
 <a name="shebang"></a>
 ### Self-Executing Scripts
 
-On UNIX systems, you can use `#!/usr/bin/env cargo eval --` as a shebang line in a Rust script.  If the script file is executable, this will allow you to execute a script file directly.
+On UNIX systems, you can use `#!/usr/bin/env -S cargo eval --` as a shebang line in a Rust script.  If the script file is executable, this will allow you to execute a script file directly.
```

Comment:
Why is the `-S` flag necessary?

Reply:
I just checked and it only seems necessary on certain linux distros (in my case arch) to feed multiple arguments into /usr/bin/env. We could revert that change or just add a disclaimer that the -S flag might be needed.

Reply:
So Linux will see `env 'cargo eval --'` and macOS will see `env cargo eval --`. `env -S cargo eval --`  will still work on macOS, so let's leave it in.